doc/command_ref.rst

```diff
@@ -1759,6 +1763,48 @@ of the most recent transaction. The last form is ``last-<offset>``, where
 ``<offset>`` is a positive integer. It specifies offset-th transaction preceding
 the most recent transaction.
 
+.. _excluded_packages-label:
+
+=================
+Excluded Packages
+=================
+
+Excluded packages are invisible for most of dnf commands and they cannot be used in transaction.
+The packages can be excluded by regular or modular filtering.
+
+-----------------
+Exclude Filtering
+-----------------
+
+It is a mechanism used by user or by DNF plugin to modify package content. Exclude filtering can be
+modified by :ref:`includepkgs <include-label>` or :ref:`excludepkgs <exclude-label>` configuration
+options in :ref:`configuration files <conf_ref-label>`. To disable excludes from configuration files
+it can be used the -:ref:`-disableexcludes <disableexcludes-label>` commandline option. Excludes
+added by a DNF plugin can be modify by the plugin interface. To disable excludes from DNF plugin it
+can be used the -:ref:`-disableplugin <disableplugin-label>` commandline option.
+
+To disable all excludes for install command following combination of options could be used:
+
+``dnf --disableexcludes=all --disableplugin=* install bash``
+
+-----------------
+Modular Filtering
+-----------------
+
+Modular filtering is a mechanism to provide only relevant content for the particular system.
+RPM packages from disabled modules or not active modular packages are excluded from DNF.
+RPM packages from active modular packages are kept on the system, but all other packages that have
+same names or provides like kept modular packages are excluded. Modular filtering is not applied
+to packages added from commandline, installed packages, or packages from repositories with
+``module_hotfixes=true`` in their .repo file.
```

Comment:
Hmm, so the modularity man page already has a section on _"Package Filtering"_. I'd start here with a reference to the modularity page and perhaps update that section with some more details. Here I'd put:

> **Please see :ref:`the modularity documentation <modularity-label>` for details on how Modular Filtering works.**
>
> **With modularity, only RPM packages from ``active`` module streams are included in the available package set. RPM packages from ``inactive`` module streams, as well as non-modular packages with the same name or provides as a package from an ``active`` module stream, are filtered out.** Modular filtering is not applied to packages added from **the command line**, installed packages, or packages from repositories with ``module_hotfixes=true`` in their .repo file.